DEPLOYMENT.md

```diff
@@ -1,49 +1,35 @@
-# Steps
+# Deployment for testing
+Comment on the PR with the deploy output, it should include environment and that it succeeded.
+Alembic revisions need to be applied from your local machine.
```

Comment:
Instructions on how this is done would be good. If someone has not worked with alembic before this can be quite cumbersome to figure out.

Reply:
Referenced Atlas.